features/eid_modifiers.lua

```diff
@@ -880,6 +880,65 @@ if EID.isRepentance then
 
 		return descObj
 	end
+	
+	-- Handle Tainted Cain pedestals
+	local function TaintedCainPedestalCallback(descObj)
```

Comment:
Please add the following changes:
- Add a Config option to disable this feature
- Change the description in a way that makes it clearer that its the "Result" of salvaging the item with TCain